devtools/conda-envs/test_env.yaml

```diff
@@ -1,6 +1,5 @@
 name: zarrtraj-test
 channels:
-  - defaults
```

Comment:
This is good and should be our default.

(Although it's probably what many people do in practice!)